resources/views/partials/forms/edit/company-select.blade.php

```diff
@@ -5,13 +5,24 @@
     {{ Form::label($fieldname, $translated_name, array('class' => 'col-md-3 control-label')) }}
         <div class="col-md-6">
             <select class="js-data-ajax" disabled="true" data-endpoint="companies" data-placeholder="{{ trans('general.select_company') }}" name="{{ $fieldname }}" style="width: 100%" id="company_select" aria-label="{{ $fieldname }}"{{ (isset($multiple) && ($multiple=='true')) ? " multiple='multiple'" : '' }}>
-                @if ($company_id = old($fieldname, (isset($item)) ? $item->{$fieldname} : ''))
-                    <option value="{{ $company_id }}" selected="selected" role="option" aria-selected="true"  role="option">
-                        {{ (\App\Models\Company::find($company_id)) ? \App\Models\Company::find($company_id)->name : '' }}
-                    </option>
-                @else
-                    <option value="" role="option">{{ trans('general.select_company') }}</option>
-                @endif
+                @isset($multiple)
+                    @if($multiple)
```

Comment:
I think there's a cleaner way to write this. See the other partials where multiple is accepted. 

https://github.com/snipe/snipe-it/blob/8001ed843676d30e33a2a4ccc4df2c930bdb2aa9/resources/views/partials/forms/edit/manufacturer-select.blade.php#L7

Notice the inline `{{ (isset($multiple) && ($multiple=='true')) ? " multiple='multiple'" : '' }}` in that example? It's a bit cleaner and we avoid having to have multiple nested layers of repeated code.

Reply:
i'll get this changed and tested. thanks for example too!